Movies/Movies/ViewControllers/SavedMoviesListViewController.swift

```diff
@@ -50,6 +55,10 @@ extension SavedMoviesListViewController {
         view = stack
     }
 
+    override func viewDidLoad() {
```

Comment:
Moved this near other view controller lifecycle methods for improved readability.